terraform/aws/variables.tf

```diff
@@ -61,10 +58,10 @@ variable "ssh_key_name" {
   default     = ""
 }
 
-variable "instance_type" {
-  description = "Instance type to use for the instance"
+variable "instance_size" {
+  description = "Instance size to use for the instance"
   type        = string
-  default     = "t3.medium"
+  default     = "micro"
```

Comment:
We have this default now defined on multiple levels. Should we keep the default in the `provision::create` plan and remove the default from here? Same applies to other defaults.